### PR TITLE
add all pet details pages (seeker/shelter)

### PR DESCRIPTION
**Pull request by JuneOh710**
(1) I'm using SVGs for the favourite icon. let me know if that's okay. 
(2) I used PX values instead of rem to follow the margin values in the figma file more closely. 
(3) Does it need to be responsive at this phase?
image

### CODE REVIEW

**Comment by iamyanbo** [comment]
The assignment details specifies that it needs to "render well on various screen sizes, including desktop monitors, tablets, and phones."
also there "should have some error message for pets in other states, i.e., pending, adopted, withdrawn."

**Comment by JuneOh710** [comment]
is there an example of "should have some error message for pets in other states, i.e., pending, adopted, withdrawn."? Is "pet unavailable" enough as an error message in those 3 cases?

**Comment by JuneOh710** [comment]
ipad-ish screen size (red text isn't actually there anymore)
image

phone screen size
image
image

**Comment by JuneOh710** [comment]
Warning state (reset to normal after delay):
image

**Comment by JuneOh710** [comment]
shelter view state:
image

shelter edit state (create state is just an empty version of this state): 
image

File: seeker/pet-details-pending.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    <link rel="stylesheet" href="../css/base.css">
+    <link rel="stylesheet" href="../css/global.css">
+    <link rel="stylesheet" href="../css/pet-details.css">
+</head>
+
+<body>
+    <div class="wrapper">
+      <header>
+        <a href="/" class="logo">
+          <img src="/img/logo_ref.png">
```
Comment (lujorryn) [inline seeker/pet-details-pending.html]: We need to change these paths to relative
```
<a href="home.html" class="logo">
          <img src="../img/logo_ref.png">
```

File: seeker/pet-details.html [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    <link rel="stylesheet" href="../css/base.css">
+    <link rel="stylesheet" href="../css/global.css">
+    <link rel="stylesheet" href="../css/pet-details.css">
+</head>
+
+<body>
+    <div class="wrapper">
+      <header>
+        <a href="/" class="logo">
+          <img src="/img/logo_ref.png">
```
Comment (lujorryn) [inline seeker/pet-details.html]: Same as above comment

File: seeker/pet-details-pending.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+                                    <path fill-rule="evenodd" clip-rule="evenodd" d="M30.5358 7.8639L27.3674 4.6955C21.1068 -1.56517 10.9562 -1.56517 4.69552 4.6955C-1.56517 10.9562 -1.56517 21.1068 4.69552 27.3675L30.4591 53.131L30.4644 53.1257L30.541 53.2024L56.3045 27.4389C62.5652 21.1782 62.5652 11.0277 56.3045 4.76703C50.044 -1.49367 39.8935 -1.49367 33.6328 4.76703L30.5358 7.8639ZM30.4644 44.8815L44.8525 30.4931L49.0813 26.4094L49.0856 26.4135L52.1823 23.3168C56.1666 19.3327 56.1666 12.8733 52.1823 8.88919C48.1984 4.90513 41.7388 4.90513 37.7549 8.88919L30.539 16.1051L30.5177 16.084L30.5148 16.0871L23.2453 8.81769C19.2612 4.8336 12.8017 4.8336 8.81768 8.81769C4.83359 12.8017 4.83359 19.2612 8.81768 23.2454L16.2196 30.6473L16.2249 30.642L30.4644 44.8815Z" fill="black"/>
+                                </svg>
+                            </div>
+                            <div class="pet-profile-info">
+                                <div class="row">
+                                    <div class="key">Shelter:</div>
+                                    <div class="value">Some Shelter</div>
```
Comment (lujorryn) [inline seeker/pet-details-pending.html]: Very minor detail but I think Yanbo used "South Shelter" as the shelter names, maybe for consistency we can change it so we can link the pages together.

**Review by lujorryn — COMMENTED** [review]
Left some small comments, otherwise the rest of the changes look good.

I noticed that your headers are not updated yet, if you want you can copy them from seeker/home.html for the seeker headers and shelter/home.html for the shelter headers, or we can clean up them all once all pages are checked in along with all the other page linking.

Note that the relative path fix needs to be applied to all of the new html files.

**Comment by iamyanbo** [comment]
image
There is a misalignment in the pet-details-create, similar with pet-details-edit, i would also add some gaps between the top 2 inputs
image
This also overlaps, maybe you can use a ul instead of nested divs
Edit: some stuff on my end was conflicting, the css is fine

**Comment by JuneOh710** [comment]
> image There is a misalignment in the pet-details-create, similar with pet-details-edit, i would also add some gaps between the top 2 inputs image This also overlaps, maybe you can use a ul instead of nested divs Edit: some stuff on my end was conflicting, the css is fine

Resolved as it was a pull error on the reviewers end.

**Comment by iamyanbo** [comment]
Could you change the href="edit-pet.html" a tag in pet-management.html in shelter to pet-details-edit.html? I originally just named it something for reference but since you made the html you can change it.

Also a nitpick, the pet name is different in pet-details and pet-details-edit, one is "Pet Name" the other is "Doggo". Otherwise looks good to me.

**Pull request merged**